src/main/java/org/zalando/nakadi/domain/EventPublishingStep.java

```diff
@@ -4,6 +4,7 @@
     NONE,
     VALIDATING,
     ENRICHING,
+    VALIDATING_SIZE,
```

Comment:
If we make it part of VALIDATING, then we will check the size of the event before enriching. I think it makes more sense to check the size after enriching, since the enrichment strategy is chose by the user who created the event type